packages/editor/src/components/browser/Browser.tsx

```diff
@@ -17,27 +17,34 @@ type BrowserProps = UseBrowserReturnValue & {
   types: BrowserType[];
   accept: (value: string, type: BrowserType) => void;
   location: string;
+  onBrowserRowDoubleClick?: () => {};
   cmsOptions?: CmsOptions;
 };
 
 const Browser = ({ open, onOpenChange, types, accept, location, cmsOptions }: BrowserProps) => {
   const { editorRef } = useEditorContext();
   const [active, setActive] = useState<BrowserType>(types[0]);
 
+  const acceptBrowser = () => {
+    accept(allBrowsers.find(browser => browser.id === active)?.accept() ?? '', active);
+  };
+
+  const onBrowserRowDoubleClick = () => {
+    onOpenChange(false);
+    acceptBrowser();
+  };
+
   const attrBrowser = useAttributeBrowser(location);
-  const cmsBrowser = useCmsBrowser(cmsOptions);
+  const cmsBrowser = useCmsBrowser(onBrowserRowDoubleClick, cmsOptions);
   const funcBrowser = useFuncBrowser();
-  const dataTypeBrowser = useDataTypeBrowser();
+  const dataTypeBrowser = useDataTypeBrowser(onBrowserRowDoubleClick);
```

Comment:
I know that you need to add this to the browsers, but honestly I think I would expect this behaviour on every browser!
Because otherwise, no one will understand why it will work on this browser but not on the other one...

Reply:
Yes, I agree with you. We can certainly brainstorm on how to solve this more effectively 👍